### ISSUE TITLE
2x assets for HiDPI displays

### ISSUE THREAD

**Issue by javy-** [tomighty/tomighty-osx#13]
Update assets for improved aesthetics.

**Comment by gianu** [comment]
:+1:

**Comment by asfaltboy** [comment]
I believe pull #10 already does this, but it seems the @ccidral did not merge them yet.

@ccidral - if you are not maintaining this anymore, are you interested in moving/forking this repo into a community user with some willing contributors to manage the PRs and issues?

**Comment by ccidral** [comment]
@asfaltboy In practice the project is inactive due to lack of time to maintain it. Anyone interested in managing the project please contact me directly to talk about it.

**Comment by asfaltboy** [comment]
Although I'm not an objective-c developer; I'll be glad to help where I can, as I am a user and have access to a Mac.


I'll try a build with some of the pull requests in the weekend and report back





Best regards,
Pavel Savchenko

**Comment by ccidral** [comment]
@asfaltboy Nice! By the way, are you a C++ developer? I had plans to rewrite Tomighty in C++ using Qt5 in an attempt to get rid of both the Java and the OS X versions. In fact, there's a basic C++ version in Tomighty's develop branch: https://github.com/ccidral/tomighty/tree/develop

**Comment by asfaltboy** [comment]
Alas @ccidral , I'm but a meager python dev who happens to own a mac. Moreover, I have overestimated my free time; only now have I gotten to do some work.

I've checked out your master, merged PRs #10 and #14 and compiled it succesfully to a smooth result.
I've skipped the other PRs (by @eduardonunesp and @dzindra) , since they deal with showing running time in status bar; which I don't really like, and no setting is offered to toggle this behavior.

As you understand, I cannot offer to take either projects directly under my wing (not enough experience with either c++/objective-c). However, seeing as many issues/PRs are still coming in, I do suggest you set up a "tomighty" organization and move all the different projects to it.

An organization would allow you to add members to the organization, create team for each project/task and administer these. Members can then request to join a team (to work on a project) and you have the flexibility of giving them admin/write/read access. For this purpose, I'm willing to administrate requests (whenever I have time) and see if I can merge it myself, otherwise involve you.
